### PR TITLE
Updates to .firefox 52.0 with New Languages

### PR DESCRIPTION
**Pull request by gobinathm**
- Including New Languages. Included language list.
— Español (de Argentina)
— Español Español (de Chile)
— Español (de España)
— Hindi
— Português (do Brasil)

### CODE REVIEW

File: Casks/firefox.rb [hunk 1]
```diff
@@ -86,4 +111,4 @@
                 '~/Library/Application Support/Firefox',
                 '~/Library/Caches/Firefox',
               ]
-end
+end
```
Comment (reitermarkus) [inline Casks/firefox.rb]: Missing newline here.

**Comment by vitorgalvao** [comment]
@gobinathm It seems you never activate the “allow edits from maintainers” checkbox. You really should, as it would make your submissions accepted faster, if we could fix the mistakes.

File: Casks/firefox.rb (hunk 1 above)
Comment (gobinathm) [inline Casks/firefox.rb]: @reitermarkus corrected it

**Comment by gobinathm** [comment]
@vitorgalvao sure will ensure its corrected in future + open pulls

**Comment by gobinathm** [comment]
@vitorgalvao FYI i just noticed, some pull requests does not have that check box enabled. Many already have it.

**Pull request merged**